### PR TITLE
Carabiners THE REVENGEANCE

### PR DESCRIPTION
**Pull request by Allspice**
#5515

Closed due to snowflake. But since then, shadowmech added the perfect solution. Works so far as I can see, aside from one strangely specific runtime. I'll do a few things later.

(also thanks to that one dude reminding me about carabiners)

:cl:
* rscadd: Adds carabiners, courtesy of Kurfurst.

### CODE REVIEW

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -816,3 +817,67 @@
...
+
+
+////
+//C-Clip
+//
+////
+
+/obj/item/device/carabiner
+	name = "carabiner"
+	desc = "A carabiner, or Karabinerhaken, is used to secure precious cargo. It is compatible with crates, air pumps, air scrubbers, space heaters, air conditioners, and canisters."
```
Comment (SonixApache) [inline code/game/objects/items/weapons/tools.dm:829]: And roller beds, and mop buckets.

**Comment by Kurfursten** [comment]
Oh baby that's a classic kurfurst sprite, which is where I would pull an image off google and downscale it with very minimal touching up. Maybe we should ask around to see if we can get a REAL spriter to do something.

**Comment by DeityLink** [comment]
Does it re-use the hookshot tether code?

**Comment by Kurfursten** [comment]
No, tethers are different than the hookshot chain as per #16335

**Comment by Pathid** [comment]
Tethering IVs is something I'd like to see. Or even tethering IVs to rollerbeds.

**Comment by Shadowmech88** [comment]
How do carabiners interact with, for example, an item that is then closed into a locker? Or anchored down? If I remember correctly, tether datums by default are ethereal and unbreakable. They'll work through walls if at a distance, for example, or will pull a locker if a tethered atom is shut inside it. Not actually sure about that last one, but I know that shutting a tethered atom inside a locker doesn't allow you to pull it away from what it's tethered to.
Just test it out and make sure that carabiners don't supernaturally affix the items to you.

**Comment by jknpj** [comment]
@Allspice when

**Comment by Kurfursten** [comment]
we just don't know

**Comment by DamianX** [comment]
cursed

**Pull request closed without merging**